common/main/root.xml

```diff
@@ -4248,6 +4248,9 @@ for derived annotations.
 			<compoundUnit type="per">
 				<compoundUnitPattern>{0}/{1}</compoundUnitPattern>
 			</compoundUnit>
+			<compoundUnit type="times">
+				<compoundUnitPattern>{0}⋅{1}</compoundUnitPattern>
+			</compoundUnit>
```

Comment:
Does the dtd also need to be updated? Currently it has
`
<!ATTLIST compoundUnit type NMTOKEN #REQUIRED >
    <!--@MATCH:literal/per-->
`
Seems like we need to expand the MATCH entry... (I am a little surprised that unit tests did not catch this)